en/components/treegrid/groupby.md

```diff
@@ -67,25 +87,58 @@ public childDataKey = "children";
 public groupColumnKey = "categories";
 ```
 
-In this sample we have also created an UI component with selector `igx-tree-grid-group-area` which handles the UI interactions related to the columns that are used for the grouping. For more information on how this component works you can take a look at the `IgxTreeGridGroupAreaComponent` class in the `tree-grid-group-area.component.ts` file. The component is completely configurable so you could copy and re-use it in your own project.
+## Angular Tree Grid Group By Load On Demand Example
 
-Here is an example of how to use the component in the template:
+<code-view style="height:850px" 
+           data-demos-base-url="{environment:demosBaseUrl}" 
+           iframe-src="{environment:demosBaseUrl}/tree-grid/treegrid-group-by-load-on-demand" alt="Angular Tree Grid Group By Load On Demand Example">
+</code-view>
+
+<div class="divider--half"></div>
+
+#### Implementation
+
+In this sample data is loaded only for a certain row on expansion. For more information on this, please refer to the [Tree Grid Load On Demand](load-on-demand.md) topic. The data is grouped by the **"ShipCountry"**, **"ShipCity"** and **"Discontinued"** fields and the resulting hierarchy is displayed in a separate column. The grouping is performed on a remote service - the data is modified and corresponding child and parent keys are assigned that are used to display the final data in a hierarchical view. For more information on how this service works you can take a look at the `TreeGridGroupingLoadOnDemandService` class in the `remoteService.ts` file.
```

Comment:
Maybe replace the first sentence with something like this: "In this sample, data is loaded in portions. Initially, the top level categories only are displayed, then child data is served once a parent row is expanded."